overlay/overrides.nix

```diff
@@ -118,6 +119,19 @@ in rec {
     };
   };
 
+  pkg-config = if pkgs.stdenv.hostPlatform == pkgs.stdenv.buildPlatform
```

Comment:
Shouldn't this be
```suggestion
  pkg-config = if pkgs.stdenv.hostPlatform != pkgs.stdenv.buildPlatform
```

Reply:
D'oh! :man_facepalming: That is precisely what it should be. Good catch there. Thank you!